src/client/js/Dialogs/Projects/ProjectsDialog.js

```diff
@@ -648,33 +620,13 @@ define([
         }
     };
 
-    ProjectsDialog.prototype._createProjectFromFile = function (projectName, jsonContent) {
-        var self = this,
-            loader = new LoaderCircles({containerElement: $('body')});
-
-        loader.start();
-
-        self._client.createProjectFromFile(projectName, null, jsonContent, self._ownerId,
-            function (err, projectId, branchName) {
-                if (err) {
-                    self._logger.error('CANNOT CREATE NEW PROJECT FROM FILE: ', err);
-                    loader.stop();
-                } else {
-                    self._logger.debug('CREATE NEW PROJECT FROM FILE FINISHED SUCCESSFULLY');
-                    self._client.selectProject(projectId, branchName, function (err) {
-                        if (err) {
-                            self._logger.error('CANNOT SELECT NEWLY CREATED PROJECT FROM FILE: ', err.message);
-                        } else {
-                            WebGMEGlobal.State.registerActiveObject(CONSTANTS.PROJECT_ROOT_ID);
-                        }
-                        loader.stop();
-                    });
-                }
-            }
-        );
-    };
-
-    ProjectsDialog.prototype._createProjectFromSeed = function (projectName, type, seedName, branchName, commitHash) {
+    ProjectsDialog.prototype._createProjectFromSeed = function (
+        projectName,
+        type,
+        seedName,
+        branchName,
+        commitHash
+    ) {
```

Comment:
is it intentional not to use the 'blobHash' parameter (it was set at the place of function call)?